### PR TITLE
New TC for [PolicyTableUpdate] Policy Table Update retry timeout defi…

### PR DESCRIPTION
**Pull request by istoimenova**
To run the script you need to replace
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForPolicyTableSnapshot.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag.lua
from #32. Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/014_ATF_P_TC_PTU_retry_timeout_definition.lua [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+function Test:TestStep_PTS_Timeout_wait_response_PTU()
+  local hmi_app_id = self.applications[config.application1.registerAppInterfaceParams.appName]
+  local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.appName } })
+  :Do(function(_,data)
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+
+      testCasesForPolicyTableSnapshot:create_PTS(true,
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/014_ATF_P_TC_PTU_retry_timeout_definition.lua]: @istoimenova create_PTS should be renamed as verify_PTS

File: test_scripts/Polices/Policy_Table_Update/014_ATF_P_TC_PTU_retry_timeout_definition.lua [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- PoliciesManager must use the values from "seconds_between_retries" section of
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/014_ATF_P_TC_PTU_retry_timeout_definition.lua]: Please change to [PolicyTableUpdate] Policy Table Update retry timeout definition

File: test_scripts/Polices/Policy_Table_Update/014_ATF_P_TC_PTU_retry_timeout_definition.lua [hunk 3]
```diff
@@ -0,0 +1,106 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- PoliciesManager must use the values from "seconds_between_retries" section of 
+-- Local PT as the values to provide in UpdateSDL request to notify SyncP manager to start PTU sequence. 
+--
+-- Description:
+-- SDL should request PTU in case user requests PTU
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/014_ATF_P_TC_PTU_retry_timeout_definition.lua]: Please check description

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**